implementation/test.py

```diff
@@ -0,0 +1,41 @@
+# KOD: Marta Rybarczyk
+
+from datetime import datetime
+from implementation.Implementation import Implementation
+
+class Test:

```

Comment:
Hm, dziwnie wyglądają te funkcje testujące.
Może spróbujesz zrobić jedną metodę, która będzie brać jako argument listę D, oraz lambd?
Wtedy, jeśli będzie miała jedną lambdę, a różne D, to jako argumenty wstawisz listę tych różnych D, i listę z jedną lambdą.

Reply:
A, i kategoria też nie powinna być argumentem?

Reply:
Oooo, i tak w ogóle
XD 
No więcej Czarek się dowal do tej klasy, no dawaj, więcej komentarzy

Możesz zrobić jedną uniwersalną metodę, o której pisałem wyżej
I wtedy kilka innych metod, które będą korzystały z tej uniwersalnej, np:
`def testForSingleLambdAndMultipleDValues(d):
          self.uniwersalnaMetoda([WARTOŚĆ_LAMBDY], d`
gdzie `d` jest listą tych wartości

Reply:
no właśnie myślałam nad tym i jeszcze się za to nie wzięłam, ale mogę ogarnąć